backend/src/main/java/ai/verta/modeldb/ModelDBConstants.java

```diff
@@ -40,6 +40,7 @@
   // feature-flags
   String DISABLED_AUTHZ = "disabled-authz";
   String STORE_CLIENT_CREATION_TIMESTAMP = "store-client-creation-timestamp";
+  String PUBLIC_SHARING_ENABLED = "public_sharing_enabled";
```

Comment:
Please this constant field in `default.config.yml` file, I think it is missing.

Reply:
done